webpack.config.js

```diff
@@ -50,106 +51,175 @@ const devtool = isProduction ? undefined : "source-map";
 /**
  * Generate a webpack configuration for a canister.
  */
-module.exports = {
```

Comment:
it looks like a lot has changed here, but it's basically just formatting. A new config was added below; everything in the old config (except for the `name` field that was added) is unchanged.